x-pack/plugins/observability_solution/slo/public/embeddable/slo/overview/slo_embeddable_factory.tsx

```diff
@@ -165,10 +165,20 @@ export const getOverviewEmbeddableFactory = ({
               const kqlQuery = groupFilters?.kqlQuery ?? '';
               const groups = groupFilters?.groups ?? [];
               return (
-                <Wrapper>
+                <div
+                  css={css`
+                    width: 100%;
+                    padding: 5px 15px;
+                    overflow: scroll;
+
+                    .euiAccordion__buttonContent {
+                      min-width: 100px;
+                    }
+                  `}
+                >
                   <EuiFlexGroup data-test-subj="sloGroupOverviewPanel" data-shared-item="">
                     <EuiFlexItem
-                      css={`
+                      css={css`
                         margin-top: 20px;
                       `}
```

Comment:
see above comment re: using EUI tokens
```suggestion
                      css={({ euiTheme }) => css`
                        margin-top: ${euiTheme.base * 1.25}px;
                      `}
```